docs/general/welcome.md

```diff
@@ -70,11 +72,14 @@ client.login('your token');
 ## Links
 * [Website](https://discord.js.org/) ([source](https://github.com/hydrabolt/discord.js-site))
 * [Documentation](https://discord.js.org/#/docs)
-* [Discord.js server](https://discord.gg/bRCvFy9)
-* [Discord API server](https://discord.gg/rV4BwdK)
+* [Discord.js Discord server](https://discord.gg/bRCvFy9)
```

Comment:
I don't see much point in putting "Discord" in these - it's pretty obvious what we're talking about.

Reply:
They are also in the GitHub Readme soo.. I will remove them